hashes/README.md

```diff
@@ -29,5 +29,5 @@ Alternatively add symlinks in your `.git/hooks` directory to any of the githooks
 
 ### Running Benchmarks
 
-We use a custom Rust compiler configuration conditional to guard the bench mark code. To run the
-bench marks use: `RUSTFLAGS='--cfg=bench' cargo +nightly bench`.
+We use a custom Rust compiler configuration conditional to guard the benchmark code. To run the
+benchmarks use: `RUSTFLAGS='--cfg=bench' cargo +nightly bench`.
```

Comment:
I think benchmark is one word.

Reply:
Right, this commit changes bench mark to benchmark..

Reply:
lol, seems I can't read diffs.